### DISCUSSION TITLE
What was the first computer to be marketed as a "laptop"?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#211] [Galaxy Trivia] [answered]
**Hint:** G SC

**Comment by Faiqlatheef** [comment, marked as answer]
The first computer to be marketed as a "laptop" was the Gavilan SC, introduced in 1983. The Gavilan SC was a portable computer that weighed around 9 pounds and had a small LCD screen built into the lid, a detachable keyboard, a built-in modem, and a battery that provided up to 8 hours of use. However, it was not as successful as later laptops due to its high price and limited functionality. The first commercially successful laptop was the Compaq Portable, introduced in 1983. It was much smaller and lighter than the Gavilan SC, and its success helped to establish the laptop as a popular computing form factor.
